lib/controller/controller.go

```diff
@@ -297,6 +307,13 @@ func (ctrl *ProvisionController) Run(stopCh <-chan struct{}) {
 	<-stopCh
 }
 
+// Set the value of failedDeletionRetryThreshold
```

Comment:
ah okay, failedRetryThreshold for provision is an argument to NewProvisonController, so never mind about SetFailedProvisionRetryThreshold. IMO we should make failedDeletionRetryThreshold an arg too right after failedRetryThreshold. I am just a bit wary about how I keep adding args to NewProvisionController and breaking it :)

I think the best thing to do would be make NewProvisionController take a struct of parameters as an argument (just like upstream PV controller) instead of like 20, and increasing, different parameters. And document each of those parameters. I will do that soon and I guess it will be a breaking change.